corehq/apps/reports/standard/deployments.py

```diff
@@ -103,6 +103,12 @@ def headers(self):
                                  help_text=_("The build profile from the user's last hearbeat request."),
                                  sortable=False)
             )
+        if self.show_version_tag:
```

Comment:
I think you need to do a similar thing in `export_table` below so that it gets a column in the excel file